### PR TITLE
Fix/rdfs list

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
Fix/rdfs list

### CODE REVIEW

File: models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php [hunk 1]
```diff
@@ -66,27 +47,13 @@ public function isApplicable(string $collectionUri): bool
...
-        $this->enrichQueryWithExcludedValueUris($searchRequest, $query);
-        $this->enrichQueryWithObjects($searchRequest, $query);
-        $this->enrichQueryWithOrderById($query);
-
+        $listClass = $this->getList($searchRequest);
         $values = [];
-        foreach ($query->execute()->fetchAll() as $rawValue) {
-            $values[] = new Value((int)$rawValue['id'], $rawValue['subject'], $rawValue['object']);
+        foreach ($listClass->getInstances() as $resource) {
+            $values[] = new Value(null, $resource->getUri(), $resource->getLabel());
```
Comment (gabrielfs7) [inline models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php:53]: The value id was here before, will it not cause any issues by removing it?

File: models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php [hunk 2]
```diff
@@ -154,15 +121,15 @@ protected function verifyUriUniqueness(Value $value): void
         }
 
         /** @noinspection PhpUnhandledExceptionInspection */
-        if ((new KernelResource($value->getUri()))->exists() || (new KernelClass($value->getUri()))->exists()) {
+        if ($this->getResource($value->getUri())->exists()) {
```
Comment (gabrielfs7) [inline models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php:124]: Why we stop checking if the class exists?

```suggestion
        if ($this->getResource($value->getUri())->exists() || $this->getClass($value->getUri())->exists()) {
```

**Review by gabrielfs7 — COMMENTED** [review]
Some doubts

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by habemuscode — APPROVED** [review]
No bump version?

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bartlomiejmarszal — COMMENTED** [review]
Just small refactoring proposal

**Pull request closed without merging**